ui/src/jobs/list/ListEnsemblingJobsLandingView.js

```diff
@@ -0,0 +1,32 @@
+import { EuiPage, EuiPanel, EuiSpacer, EuiText } from "@elastic/eui";
+import React from "react";
+import { useConfig } from "../../config";
+import { ListEnsemblingJobsView } from "./ListEnsemblingJobsView";
+
+export const ListEnsemblingJobsLandingView = (props) => {
+  const {
+    appConfig: { batchEnsemblingEnabled },
+  } = useConfig();
+
+  return (
+    <>
+      {batchEnsemblingEnabled ? (
+        <ListEnsemblingJobsView {...props} />
+      ) : (
+        <EuiPage>
+          <EuiPanel>
+            <EuiText>
+              Batch ensembling has not been enabled for this deployment of
+              Turing.
+            </EuiText>
+            <EuiSpacer />
+            <EuiText>
+              To use batch ensembling, please enable it in the configuration
+              file used to deploy the Turing API.
```

Comment:
Nit: Can it be "Turing app" instead of "Turing API" ? It needs to be enabled both on the API and the UI, right? Besides, saying "API" in the UI view feels too technical.

Reply:
That's true; I've edited the text to reflect "Turing app" instead as you suggested :)